### PR TITLE
[hi] Add content/hi/docs/home/_index.md

### PR DESCRIPTION
**Pull request by verma-kunal**
This PR is a part of ongoing `hi` localization effort

/language hi

### CODE REVIEW

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:2]:
```suggestion

```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:3]:
```suggestion

```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Comparaing with `ko` localization, I found we can remove `approvers:`

File: content/hi/docs/home/_index.md
Comment (verma-kunal) [reply content/hi/docs/home/_index.md:2]: oh yes!! I saw this & had left that for the last !!
Thanks for pointing it out 👍

File: content/hi/docs/home/_index.md
Comment (verma-kunal) [inline content/hi/docs/home/_index.md:7]: @anubha-v-ardhan needed your suggestions here!! what can `Home` be replaced with in Hindi
I was a bit confused in that

Comment (anubha-v-ardhan) [reply content/hi/docs/home/_index.md:7]: `होम`, the `ko` localization has used `홈` which is basically the exact sound of `home`, just written in their respective script. We can do the same for these type of problems.

Comment (verma-kunal) [reply content/hi/docs/home/_index.md]: alright! done that 👍

File: content/hi/docs/home/_index.md
Comment (mittalyashu) [inline content/hi/docs/home/_index.md:13]:
```suggestion
    title: "प्रलेखन"
```

Comment (mittalyashu) [reply content/hi/docs/home/_index.md:13]: I am not sure if they changed their docs guidelines recently, but you can also try removing the `"`.

**Comment by verma-kunal** [comment]
@mittalyashu , actually i was referring the `Korean` Localization & according to that did the necessary changes

**Comment by mittalyashu** [comment]
After taking a took at english doc, I think it's best to remove the quotes, as it will make the content readable more easily from codebase.

**Comment by kodeninja** [comment]
IMO, "कुबेरनेट्स" does not match `kubernetes` (in spelling or pronunciation). Perhaps, something like "कूबरनेटिस"?

Also, it would be great if we could at least _attempt_ to provide the pure Hindī terms for things like "container", "pod", "engine", "orchestration" and so on, rather than just writing the english words in dévanāgarī.

Hopefully, someone with enough Sanskrit knowledge can help come up with appropriate terms for, at least, the important concepts 🤷🏼‍♂️.

**Comment by verma-kunal** [comment]
> After taking a took at english doc, I think it's best to remove the quotes, as it will make the content readable more easily from codebase.

which quotes are you referring to here? can you pls point that out !!

**Comment by verma-kunal** [comment]
> IMO, "कुबेरनेट्स" does not match `kubernetes` (in spelling or pronunciation). Perhaps, something like "कूबरनेटिस"?
> 
> Also, it would be great if we could at least _attempt_ to provide the pure Hindī terms for things like "container", "pod", "engine", "orchestration" and so on, rather than just writing the english words in dévanāgarī.
> 
> Hopefully, someone with enough Sanskrit knowledge can help come up with appropriate terms for, at least, the important concepts 🤷🏼‍♂️.
> 
> https://www.learnsanskrit.cc/ can perhaps be used to suggest some options.

I get your point there! Maybe we can find certain words & add those to the sheet here: https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0
for ease of use!!

**Comment by kodeninja** [comment]
@verma-kunal, thanks! I added some comments to that google doc, and also sent a request to become editor, if that's ok.

Also, I added some terms in the main PR under https://github.com/kubernetes/website/issues/29353#issuecomment-898740447. You can add those to the doc as well.

I follow some twitter handles like [@vakibs](https://twitter.com/vakibs) who are polyglots, and very knowledgeable in Indic languages. We can take their help/guidance in coming up the appropriate Hindī terms for the core concepts.

The best part is that the same terms, since they come from Sanskrit, can be used in translations in other Indic languages as well.

**Comment by anubha-v-ardhan** [comment]
> IMO, "कुबेरनेट्स" does not match `kubernetes` (in spelling or pronunciation). Perhaps, something like "कूबरनेटिस"?
> 
> Also, it would be great if we could at least _attempt_ to provide the pure Hindī terms for things like "container", "pod", "engine", "orchestration" and so on, rather than just writing the english words in dévanāgarī.
> 
> Hopefully, someone with enough Sanskrit knowledge can help come up with appropriate terms for, at least, the important concepts 🤷🏼‍♂️.
> 
> https://www.learnsanskrit.cc/ can perhaps be used to suggest some options.

@kodeninja We appreciate your suggestions but in my view the Kubernetes objects should be written the same as in English, just in Devnagri script. The goal of localization is to help folks understand the docs in Hindi and not word to word pure translation. Hope you understand.

**Comment by verma-kunal** [comment]
> @verma-kunal, thanks! I added some comments to that google doc, and also sent a request to become editor, if that's ok.
> 
> Also, I added some terms in the main PR under [#29353 (comment)](https://github.com/kubernetes/website/issues/29353#issuecomment-898740447). You can add those to the doc as well.
> 
> I follow some twitter handles, like [@vakibs](https://twitter.com/vakibs), who are polyglots, and very knowledgeable in Indic languages. We can take their help/guidance in coming up the appropriate Hindī terms for the core concepts.
> 
> The best part is that the same terms, since they come from Sanskrit, can be used in translations in other Indic languages as well.

As mentioned by @anubha-v-ardhan , the aim of this localization is not to have `pure word to word translation` of all the terms but to have the docs in Hindi for ease of understanding!! 
Just simple translations would do the work as well, Hope you understand!!
Thank you for the suggestions & feedbacks. Would love to see you around contributing to the initiative 🙂

File: content/hi/docs/home/_index.md
Comment (sftim) [inline content/hi/docs/home/_index.md:56]:
```suggestion
  button_path: /docs/contribute
```
unless `/hi/docs/contribute` is already there to link to.

File: content/hi/docs/home/_index.md
Comment (sftim) [inline content/hi/docs/home/_index.md:41]: To make this work, you'll need to localize `/docs/tasks` (which is OK, but the link target needs to exist).

File: content/hi/docs/home/_index.md
Comment (sftim) [inline content/hi/docs/home/_index.md:60]: Perhaps use डाउनˈलअउड्/ instead of download?

**Review by sftim — COMMENTED** [review]
Wow, there's good steps being made here. Here's some feedback on what I see at the moment.

File: content/hi/docs/home/_index.md
Comment (verma-kunal) [reply content/hi/docs/home/_index.md:56]: Oh, actually i was following the `Korean` localization, so according to that i edited the link!
Sure, we can remove that for now 🙂

File: content/hi/docs/home/_index.md
Comment (verma-kunal) [reply content/hi/docs/home/_index.md:41]: we'll be localization that file as well, right, in the near future !! So, for the time-being, should we remove this or keep that as it is (again, taking ref. from `Korean `localization), but i do get your point here 🤔

File: content/hi/docs/home/_index.md
Comment (verma-kunal) [reply content/hi/docs/home/_index.md]: Done! Localized the whole text now

**Comment by mittalyashu** [comment]
@verma-kunal Can you remove double quotes from the frontmatter in markdown?

**Comment by verma-kunal** [comment]
> @verma-kunal Can you remove double quotes from the frontmatter in markdown?

@mittalyashu can you pls specify like from which parts I have to remove the `double quotes`!! that would be helpful

File: content/hi/docs/home/_index.md
Comment (mittalyashu) [inline content/hi/docs/home/_index.md:60]:
```suggestion
  button: डाउनलोड कुबेरनेट्स
```

for example

File: content/hi/docs/home/_index.md
Comment (sftim) [reply content/hi/docs/home/_index.md:41]:
```suggestion
  button_path: "/docs/tasks"
```
_for now_

File: content/hi/docs/home/_index.md
Comment (sftim) [reply content/hi/docs/home/_index.md:60]: Either is fine; YAML doesn't mind.

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:29]:
```suggestion
  description: "कुबेरनेट्स में एप्लिकेशन को डिप्लॉय करने के लिए ट्यूटोरियल का अनुसरण करें|"
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:33]:
```suggestion
  title: "एक कुबेरनेट्स क्लस्टर सेटअप करें"
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:35]:
```suggestion
  button: "कुबेरनेट्स सेटअप करें "
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:39]:
```suggestion
  description: "चरणों के एक छोटे क्रम का अनुसरण करके सामान्य कार्यों को करें।"
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:44]:
```suggestion
  description: "कुबेरनेट्स में प्रमाणित हो कर अपने क्लाउड नेटिव प्रोजेक्ट सफल बनाएं!"
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:49]:
```suggestion
  description: शब्दावली, कमांड लाइन सिंटैक्स, API संसाधनो के प्रकार और टूल सेटअप करने के प्रलेखन। 
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:51]:
```suggestion
  button_path: /docs/reference
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:54]:
```suggestion
  description: कोई भी योगदान दे सकता है, चाहे आप प्रोजेक्ट में नए हैं या आप लंबे समय से हैं|
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:58]:
```suggestion
  title: कुबेरनेट्स रिलीज नोट्स 
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:59]:
```suggestion
  description: यदि आप कुबेरनेट्स इंस्टॉल कर रहे हैं या नवीनतम संस्करण में अपग्रेड कर रहे हैं, तो वर्तमान रिलीज़ नोट्स देखें|
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:60]:
```suggestion
  button: "कुबेरनेट्स डाउनलोड करें"
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Final Review : )
Please point out if I have still missed some inaccuracy.

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:64]:
```suggestion
  description: इस वेबसाइट में कुबेरनेट्स के वर्तमान और पिछले 4 संस्करणों के लिए प्रलेखन हैं|
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/home/_index.md
Comment (verma-kunal) [reply content/hi/docs/home/_index.md:29]: i think till then, we had not decided a global word for this!! so, thats why wasn't able to make changes
Thanks

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [reply content/hi/docs/home/_index.md:33]: cc: @verma-kunal
If you agree with the changes, please commit these suggestions :)

**Comment by sftim** [comment]
/label tide/merge-method-squash

Feel free to remove this label (or request removal) if you squash those commits yourself, @verma-kunal

**Comment by sftim** [comment]
When that label is set, Prow tells GitHub to squash the PR's commits onto the target branch, rather than doing a merge.

It's still slightly tidier if the author squashes their commits, rather than setting the label and leaving it to GitHub. Nevertheless, using the label-based squashing is fine.

**Comment by anubha-v-ardhan** [comment]
Got it, Thanks!

**Comment by verma-kunal** [comment]
> When that label is set, Prow tells GitHub to squash the PR's commits onto the target branch, rather than doing a merge.
> 
> It's still slightly tidier if the author squashes their commits, rather than setting the label and leaving it to GitHub. Nevertheless, using the label-based squashing is fine.

Oh! got it!! So, in the future PRs, should we follow the label-squashing approach or user squashing their commits?

**Comment by verma-kunal** [comment]
> /label tide/merge-method-squash
> 
> Feel free to remove this label (or request removal) if you squash those commits yourself, @verma-kunal

got it!!

**Comment by anubha-v-ardhan** [comment]
@verma-kunal Author squashing commit is tidier, so we will prefer it.

**Comment by verma-kunal** [comment]
Squashed the commits!! Please do have a look 🙂

remove /label tide/merge-method-squash

**Comment by verma-kunal** [comment]
remove tide/merge-method-squash

**Comment by verma-kunal** [comment]
remove-tide/merge-method-squash

**Comment by anubha-v-ardhan** [comment]
@verma-kunal Some suggestions were pending to be incorporated. We generally squash commits once the PR is finalised to be merged. 
However, you can implement the suggestions in one more commit, we can merge 2 commits without squash.

**Comment by verma-kunal** [comment]
> @verma-kunal Some suggestions were pending to be incorporated. We generally squash commits once the PR is finalised to be merged.
> Can you look at the suggestions and implement the changes.

i actually commited those changes manually!! can you please check?

**Comment by anubha-v-ardhan** [comment]
The changes seem to be hidden due to too many conversations. You can go to files changed tab to see them. There are suggestions left

**Comment by verma-kunal** [comment]
> The changes seem to be hidden due to too many conversations. You can go to files changed tab to see them. There are suggestions left

sure, would check these!!

File: content/hi/docs/home/_index.md
Comment (verma-kunal) [reply content/hi/docs/home/_index.md]: done manually!!

File: content/hi/docs/home/_index.md
Comment (verma-kunal) [reply content/hi/docs/home/_index.md]: done manually !!

**Comment by verma-kunal** [comment]
@anubha-v-ardhan , is this good to go? 🙂

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:24]:
```suggestion
  description: "कुबेरनेट्स और इसकी मूलभूत अवधारणाओं के बारे में जानें।"
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:29]:
```suggestion
  description: "कुबेरनेट्स में एप्लिकेशन को डिप्लॉय करने के लिए ट्यूटोरियल का अनुसरण करें।"
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:34]:
```suggestion
  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें।"
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:54]:
```suggestion
  description: कोई भी योगदान दे सकता है, चाहे आप प्रोजेक्ट में नए हैं या आप लंबे समय से हैं।
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:59]:
```suggestion
  description: यदि आप कुबेरनेट्स इंस्टॉल कर रहे हैं या नवीनतम संस्करण में अपग्रेड कर रहे हैं, तो वर्तमान रिलीज़ नोट्स देखें।
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:64]:
```suggestion
  description: इस वेबसाइट में कुबेरनेट्स के वर्तमान और पिछले 4 संस्करणों के लिए प्रलेखन हैं।
```

**Review by anubha-v-ardhan — COMMENTED** [review]
Please note that `।` and `|` are different characters and we should prefer `।`

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:20]:
```suggestion
  कुबेरनेट्स कंटेनरीकृत एप्लीकेशन के डिप्लॉयमेंट, स्केलिंग और प्रबंधन को स्वचालित करने के लिए एक ओपन सोर्स कंटेनर ऑर्केस्ट्रेशन इंजन है। यह ओपन सोर्स प्रोजेक्ट Cloud Native Computing Foundation द्वारा होस्ट किया गया है|(<a href="https://www.cncf.io/about">CNCF</a>).
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Review by anubha-v-ardhan — COMMENTED** [review]
Can you squash the commits please so we can merge this.

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:34]:
```suggestion
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:36]:
```suggestion
```

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:38]:
```suggestion
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/home/_index.md
Comment (anubha-v-ardhan) [inline content/hi/docs/home/_index.md:40]:
```suggestion
- name: setup
  title: "एक कुबेरनेट्स क्लस्टर सेटअप करें"
  description: "अपने संसाधनों और जरूरतों के आधार पर कुबेरनेट्स को चालू करें।"
  button: "कुबेरनेट्स सेटअप करें "
  button_path: "/hi/docs/setup"
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/home/_index.md
Comment (verma-kunal) [reply content/hi/docs/home/_index.md:40]: i wonder where that all came from? 🤔

**Review by anubha-v-ardhan — COMMENTED** [review]
LGTM
You can squash the commits now!

**Comment by verma-kunal** [comment]
> LGTM
> You can squash the commits now!

there are only 2 commits after the previous squash!! do we need still need to do a squash here?

**Comment by anubha-v-ardhan** [comment]
Sorry, but seems like we have 16 commits here. Maybe the previous squash failed or for some reason.

**Comment by anubha-v-ardhan** [comment]
We can merge using `tide/merge-method-squash` if there's an issue.

**Comment by verma-kunal** [comment]
> We can merge using `tide/merge-method-squash` if there's an issue.

no worries at all !! i can do that again 🙂

**Comment by verma-kunal** [comment]
Done!! is this good to go?

**Comment by anubha-v-ardhan** [comment]
/remove-tide/merge-method-squash

**Comment by anubha-v-ardhan** [comment]
/lgtm
/approve

Thank you @verma-kunal

**Pull request merged**
